nifi/Dockerfile

```diff
@@ -89,5 +66,55 @@ COPY shared/log4shell_scanner /bin/log4shell_scanner
 RUN /bin/log4shell_scanner s /stackable/nifi-${PRODUCT}
 # ===
 
+# Final image
+FROM stackable/image/java-base
+
+ARG PRODUCT
+ARG RELEASE
+
+LABEL name="Apache NiFi" \
+      maintainer="info@stackable.tech" \
+      vendor="Stackable GmbH" \
+      version="${PRODUCT}" \
+      release="${RELEASE}" \
+      summary="The Stackable image for Apache NiFi." \
+      description="This image is deployed by the Stackable Operator for Apache NiFi."
+
+RUN microdnf update && \
+    microdnf install \
+    python-devel \
```

Comment:
is this really needed?

Reply:
Did not check was in the main image before.

Reply:
The python stuff is in there for the nypi api (which is used by the reporting task). This could go into a lightweight docker image on its own (i think we had this a while ago but decided to use one image (multiarch?)).
Since NiFi 2.0 offers heavy python support i think we will have it in there anyways at that point.
Opinions @lfrancke @razvan ?

Reply:
`python-devel` is just C headers. I'm 99% sure it's not needed but it doesn't do much damage either if we don't want to risk removing it

Reply:
yeah just tested and could remove a couple of dependencies https://github.com/stackabletech/docker-images/pull/678/commits/ff1f63c7fe39aec7d5b4583e51d77aa4cdb9d445